src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -207,125 +207,159 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         }
     }
 
+    handleSaveSingleGiftEntry(sectionsList,enableSave,toggle) {
+
+        // handle error on callback from promise
+        const handleCatchError = (err) => this.handleCatchOnSave(err);
+
+        GeFormService.handleSave(sectionsList, this.donorRecord, this.blankDataImportRecord).then(opportunityId => {
+            this.navigateToRecordPage(opportunityId);
+        }).catch(error => {
+            enableSave();
+            toggle();
+            handleCatchError(error);
+        });
+
+    }
+
+    handleSaveBatchGiftEntry(sectionsList,enableSave,toggle) {
+
+        // reset function for callback
+        const reset = () => this.reset();
```

Comment:
Previously it looks like we were resetting the form for both Single and Batch gift entry - with this change it's only happening for batch. Is that intentional?

Reply:
yes, previously It was only being used for BGE mode